cyral/assets/dashboards/cyral_overview.json

```
@@ -0,0 +1,265 @@
+{
+    "author_info": {
+        "author_name": "Cyral"
+    },
+    "board_title": "Cyral",
+    "description": "",
```

Can we add a description about this dashboard?

```suggestion
    "description": "The default dashboard for monitoring Cyral MySQL sidecars, including policies, query latency and durations, and table metrics.",
```